CONTRIBUTING.md

```diff
@@ -95,35 +78,35 @@ $ git add my/changed/files
 $ git commit
 ```
 
-### Commit guidelines
-
-Writing good commit logs is important. A commit log should describe what
-changed and why. Follow these guidelines when writing one:
```

Comment:
I would suggest leave this line, and ref to where ever #12744 / #12791 lands
IMHO we should hoist this section one level to \## but take it out of the "workflow"
Move the whole "\### commit guidelines" to Just before \## Additional Notes.